example/org.eclipse.glsp.ide.workflow.editor/plugin.xml

```diff
@@ -88,93 +89,6 @@
              schemeId="org.eclipse.ui.defaultAcceleratorConfiguration"
              sequence="A">
        </key>
-    </extension>
-    <extension
```

Comment:
:+1:

I think we should remove the Handler classes as well. Global Action Handlers are much more appropriate for these actions

Reply:
I really thought I did so thank you for catching that!